.github/workflows/push.yml

```diff
@@ -211,9 +211,10 @@ jobs:
         MDB: postgresql://postgres:postgres@0.0.0.0:5432/metadata_%s?min_size=2&max_size=8
         PDB: postgresql://postgres:postgres@0.0.0.0:5432/precomputed_%s?min_size=2&max_size=6
         RDB: postgresql://postgres:postgres@0.0.0.0:5432/persistentdata_%s?min_size=2&max_size=5
+        MDB_EPHEMERAL: postgresql://postgres:postgres@0.0.0.0:5432/metadata_ephemeral_%s?min_size=2&max_size=8
       run: |
         set -x
-        for db in state metadata precomputed persistentdata; do
+        for db in state metadata precomputed persistentdata metadata_ephemeral; do
```

Comment:
Can you please avoid this ephemeral change or at least move to a separate PR, because it needs a further discussion. I am against this overcomplication, for example.

Reply:
My target here was to have mdb in tests that is empty and writable